src/app/component/shared/action-menu/action-menu.component.spec.ts

```diff
@@ -93,7 +93,7 @@ describe("ActionMenuComponent", () => {
           actions: List<AnyMenuItem>([]),
           links: List<NavigableMenuItem>([]),
         },
-      } as PageInfoInterface);
+      } as IPageInfo);
```

Comment:
These type conversions are unnecessary, but can be fixed when the tests are re-written someday